src/main/kotlin/no/nav/sifinnsynapi/k9ettersending/K9EttersendingBeskjedProperties.kt

```diff
@@ -0,0 +1,32 @@
+@file:Suppress("ConfigurationProperties")
+
+package no.nav.sifinnsynapi.k9ettersending
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+import org.springframework.validation.annotation.Validated
+
+interface K9EttersendingBeskjedProperties{
```

Comment:
Hvis du tenker å ha flere implementasjon av denne interfacen så bør du kalle den BeskjedProperties og la andre BeskjedProperties implementere den.

Reply:
Da tenker jeg på i tillegg til `K9EttersendingPPBeskjedProperties ` og `K9EttersendingOMSBeskjedProperties `

Reply:
Ja tenker iallfall vi burde gjøre det når vi gjør en større refaktorering. Nå har nesten alle ytelsene en egen "tilK9Beskjed" metode, men det kunne vært en felles. Den metoden kunne tatt inn Keys som argument, så tar vi høyde for at noen har "soker" og andre har "søker".